### DISCUSSION TITLE
Show areas of reachability based on time or distance

### DISCUSSION THREAD

**Discussion by fragosus** [mapcentia/vidi#254] [Ideas]
Hi, 

Is there any chance to implement the "**leaflet.reachability**" plugin into this application? As an extra feature, the app could list the facilities have reached. (So the result polygon can be worked as an advanced query.)

https://github.com/traffordDataLab/leaflet.reachability

**Comment by mapcentia** [comment]
I guess you could bundle the plugin up and activate it. But there could be some issues with GUI integration
